test/cpp/jit/test_gpu.cpp

```diff
@@ -15415,6 +15415,215 @@ TEST(NVFuserTest, TestBackOffInnerBroadcast3_CUDA) {
   TORCH_CHECK(tv3->getMaxProducerPosition() == 3);
 }
 
+TEST(NVFuserTest, TestSimpleWarp_CUDA) {
+  auto fusion = std::make_unique<Fusion>();
```

Comment:
Any reason naming the test as `Test..._CUDA` rather than `Fusion..._CUDA`?

Reply:
Updated thanks!